### PR TITLE
fix(decdk): relax validation when not using constructs

### LINKED ISSUE #1986: decdk: Can't apply intrinsic functions to raw CloudFormation
Therefore, it is not a `can be used as-is` condition as written in the [document](https://github.com/awslabs/aws-cdk/tree/v0.25.2/packages/decdk#raw-cloudformation).

If this becomes possible, will it be possible to deploy existing CloudFormation templates with CDK?

### PR DESCRIPTION
**Pull request by eladb**
Remove all CloudFormation resources from base JSON schema so basically
CFN resources (identified by having "::" in their resource type)
are "passed through" as JSON blobs.

Fixes #1986

----

### Pull Request Checklist

* [x] Testing
  - Unit test added (prefer not to modify an existing test, otherwise, it's probably a breaking change)
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [x] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
* [x] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [x] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: packages/decdk/README.md [hunk 1]
```diff
@@ -269,8 +271,7 @@ deCDK can deconstruct APIs that adhere to the standards defined by __awslint__ a
 
 As you might have observed, whenever users need to reference a type in deCDK templates they are required to reference the fully qualified name (e.g. `@aws-cdk/aws-s3.Bucket`). We can obvsiouly come up with a more concise way to reference these types, as long as it will be possible to deterministically translate back and forth.
 
-### Misc
+### Special Types
 
-- `iam.PolicyDocument` is be tricky since it utilizes a fluent API. We need to think whether we want to revise the PolicyDocument API to be more compatible or add a utility class that can help.
+`iam.PolicyDocument` is be tricky since it utilizes a fluent API. We need to think whether we want to revise the PolicyDocument API to be more compatible or add a utility class that can help.
```
Comment (rix0rrr) [inline packages/decdk/README.md]: is be

File: packages/decdk/README.md [hunk 2]
```diff
@@ -192,6 +192,8 @@ Outputs:
 
 If deCDK doesn't identify a resource type as a CDK resource, it will just pass it through to the resulting output. This means that any existing CloudFormation/SAM resources (such as `AWS::SQS::Queue`) can be used as-is.
 
+The decdk JSON schema will simply pass through any resources that have a type that includes `::`, so don't expect any validation of raw CloudFormation resource properties.
```
Comment (RomainMuller) [inline packages/decdk/README.md]: But... but we could perform validation, couldn't we?

Comment (eladb) [reply packages/decdk/README.md]: We could, but we don't at this point.

Comment (eladb) [reply packages/decdk/README.md]: https://github.com/awslabs/aws-cdk/issues/2001

**Pull request merged**